README.md

```diff
@@ -31,10 +23,22 @@ We welcome contributions from everyone! If you are interested in contributing, o
 
 We follow the [Code of Conduct](CODE_OF_CONDUCT.md) to ensure a safe and respectful environment for all. We expect all community members to abide by these guidelines. If you witness any violations, please contact the project maintainers.
 
+## Community Meetings
+
+We hold bi-weekly community meetings to promote communication and collaboration within our community. These meetings are organized and hosted by Maintainers to synchronize progress, resolve issues, and plan for future development. All contributors and anyone interested in the Jeandle project are welcome to participate.
+
+For detailed information about meeting schedules, agenda, and participation guidelines, please see our [Community Meeting Guide](MEETING.md).
+
 ## Amending this Repository
 
 Any significant changes to this governance repository will be discussed among the maintainer team and require a vote to pass. For minor adjustments and clarifications, maintainers may make changes after reaching a general consensus.
 
+## Security Reporting
+
+If you discover a security vulnerability in the Jeandle project, please report it privately to our security team. We take security issues seriously and will respond promptly to address any concerns.
+
+To report a security issue, please contact the project maintainers directly through our community channels or email the maintainers listed in [MAINTAINERS.md](MAINTAINERS.md).
+
```

Comment:
We use Github security advisory to report vulnerabilities. Refer to https://github.com/jeandle/jeandle-jdk/blob/main/SECURITY.md